modules/supported-installation-methods.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * configuration/internal-certificate-authority-rotation-for-rhacs.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="supported-installation-methods_{context}"]
+= Supported installation methods
```

Comment:
This is not the most accurate heading - it's really CA rotation support per installation method, so I would suggest something like:

CA rotation support for different installation methods
CA rotation support per installation method
CA rotation support by installation